api/base/utils.py

```diff
@@ -16,8 +15,7 @@ def absolute_reverse(view_name, query_kwargs=None, args=None, kwargs=None):
     """Like django's `reverse`, except returns an absolute URL. Also add query parameters."""
     relative_url = reverse(view_name, kwargs=kwargs)
 
-    url = website_util.api_v2_url(relative_url, params=query_kwargs,
-                                  base_prefix=api_settings.API_PATH)
+    url = website_util.api_v2_url(relative_url, params=query_kwargs, base_prefix='')
```

Comment:
Since the function call signature is being changed to use `base_prefix=''` as the default, there's no need for that argument here.